earthfile2llb/local_state_cache_escape.go

```diff
@@ -0,0 +1,50 @@
+package earthfile2llb
+
+import (
+	"unicode/utf8"
+)
+
+// This file is based on regexp.QuoteMeta and helper functions, but has been adapted
```

Comment:
So buildkit also ported some functions from regexp.QuoteMeta?

Reply:
nope, buildkit passes this copy string to https://pkg.go.dev/path/filepath#Match and doesn't provide any escaping.